lib/bolognese/readers/datacite_reader.rb

```diff
@@ -125,11 +125,15 @@ def read_datacite(string: nil, **options)
 
         dates = Array.wrap(meta.dig("dates", "date")).map do |r|
           if r.is_a?(Hash) && date = sanitize(r["__content__"]).presence
-            if Date.edtf(date).present? || Bolognese::Utils::UNKNOWN_INFORMATION.key?(date)
-              { "date" => date,
-                "dateType" => parse_attributes(r, content: "dateType"),
-                "dateInformation" => parse_attributes(r, content: "dateInformation")
-              }.compact
+            begin
+              if Date.edtf(date).present? || Bolognese::Utils::UNKNOWN_INFORMATION.key?(date) || Time.iso8601(date).present?
+                { "date" => date,
+                  "dateType" => parse_attributes(r, content: "dateType"),
+                  "dateInformation" => parse_attributes(r, content: "dateInformation")
+                }.compact
+              end
+            rescue
```

Comment:
This is my first time seeing `rescue nil`?
Does this catch all exceptions?  That might be too broad.  If there are specific types of exceptions we need to guard against, it could be good to specify the type.

If this was more about providing a default value, should will be returned if the `if` statement fails.

Reply:
Yes rescue nil is very broad, a quick google also suggests some articles it's quite slow.
What's the actual exception being thrown and from what bit? The time.iso8601?

Reply:
Yes, the rescue is for the Time.iso8601 method. It throws errors if the date format is wrong, whereas the Date.edtf method doesn't. I can narrow down the rescue to more specific exceptions though